shared/chat/conversation/messages/message-popup/reactionitem.tsx

```diff
@@ -50,7 +50,6 @@ const styles = Kb.Styles.styleSheetCreate(
         justifyContent: 'space-between',
         paddingLeft: Kb.Styles.globalMargins.small,
         paddingRight: Kb.Styles.globalMargins.small,
-        paddingTop: Kb.Styles.globalMargins.tiny,
```

Comment:
i added this to fix an offset but its actually an issues w/ the item above so the fix is moved there